app/javascript/dashboard/components/layout/Sidebar.vue

```diff
@@ -125,6 +131,9 @@ export default {
     shouldShowInboxes() {
       return this.sidemenuItems.common.routes.includes(this.currentRoute);
     },
+    shouldShowTeams() {
+      return this.shouldShowInboxes && this.teams.length;
```

Comment:
I see `shouldShowInboxes` used for showing labels too, @pranavrajs is this generic condition which is named wrong ?